forth_code/README.md

```diff
@@ -1,5 +1,6 @@
-# Raw Forth flies for Tali Forth 2 for the 65c02  
-Scot W. Stevenson <scot.stevenson@gmail.com>   
+# Raw Forth flies for Tali Forth 2 for the 65c02
```

Comment:
I never noticed the Raw Forth **flies** before - if you don't fix it, I'll add it to my documentation errors list and get it fixed.